cmd/server/main.go

```diff
@@ -0,0 +1,70 @@
+package main
+
+import (
+	"fmt"
+	"net/http"
+	"os"
+	"os/signal"
+	"syscall"
+
+	"github.com/go-kit/kit/log"
+	"github.com/redhatinsights/insights-operator-conditional-gathering/pkg/service"
+	"github.com/redhatinsights/insights-operator-conditional-gathering/pkg/service/transport"
+)
+
+const (
+	httpAddr = ":8080"
+)
+
+func main() {
+	// Logger
+	var logger log.Logger
+	logger = log.NewLogfmtLogger(log.NewSyncWriter(os.Stderr))
+	logger = log.With(logger, "ts", log.DefaultTimestampUTC)
+
+	// Repository
+	dataPath := os.Getenv("RULES_DATA_PATH")
+	repo := service.NewRepository(dataPath)
+
+	// Services
+	svc := service.New(repo)
+
+	httplogger := log.With(logger, "component", "http")
+
+	mux := http.NewServeMux()
+	// Control CORs and other stuff
+	http.Handle("/", accessControl(mux))
+
+	// Handlers
+	mux.Handle("/", transport.NewHTTPHandler(svc, httplogger))
```

Comment:
These two lines above look suspicious to me. Shouldn't we rather set the access control for the mux handler?